src/nyaml/nyaml2nxdl.py

```diff
@@ -1155,6 +1175,8 @@ def nyaml2nxdl(input_file: str, out_file, verbose: bool):
     schema, definitions then evaluates a nested dictionary of groups recursively and
     fields or (their) attributes as children of the groups
     """
+    nxdl_copyright_license = get_nxdl_copyright_license(nxdl_file=out_file)
```

Comment:
This is calling the function with the name of output file, which may not even exists. 
In case of nxdl->nyaml conversion one could call this function with the input file name. In case of nyaml->nxdl conversion the copyright should be in the nyaml file as part of the preserved nxdl conent.